README.md

```diff
@@ -33,10 +33,9 @@
 
 # Getting started
 
-### Script 
+### NewProjectCI 
 1. Download this repository extract and open the folder
-2. Run `newproject.sh` script to create a copy with the name you want 
-    * Example: `./newproject.sh -p com.example.project -t Project` 
+2. Run `java -jar NewProjectCI` script to create a copy with the name you want
```

Comment:
Do we really need a `jar`, can't we compile and run just in time? Just so users don't have to trust a random "executable".